### DISCUSSION TITLE
How can I tweak this to make all backgrounds black?

### DISCUSSION THREAD

**Discussion by ngeraks** [rafaelmardojai/firefox-gnome-theme#284] [Q&A] [answered]
How can I tweak this to make all backgrounds black like my terminal?
![image](https://user-images.githubusercontent.com/78253750/133258953-0be46196-8b2b-4d29-9d61-3c50fb77da6d.png)

**Comment by rafaelmardojai** [comment, marked as answer]
Try editing the color variables CSS files located in `theme/colors/dark.css` and `theme/colors/light.css`, and set the same color on all the variables with `-background` and ` -border-color`.

**Comment by ngeraks** [comment]
Thank you. That did it.
![Screenshot from 2021-09-23 15-05-51](https://user-images.githubusercontent.com/78253750/134504263-ab7a65f8-9d1d-4548-b45b-8710c0b18003.png)
~/.mozilla/firefox/.../chrome/firefox-gnome-theme/theme/colors/dark.css
[dark.css.txt](https://github.com/rafaelmardojai/firefox-gnome-theme/files/7217423/dark.css.txt)
